webdriver-spec.html

```diff
@@ -3039,16 +3040,18 @@
 <section>
 <h2>Element Retrieval</h2>
 
-<p>The <a>Find Element</a> and <a>Find Elements</a> <a>commands</a>
- allows look up of individual and a collection of <a>elements</a>, respectively.
+<p>The <a>Find Element</a> and <a>Find Elements</a> commands
+ allow look up of individual- and collections of <a>elements</a>.
+ Element retrieval searches are performed
```

Comment:
You have not tied the term 'Element Retrieval' to a definition/command. So this does not follow from the previous sentence.